src/pages/Login/Login.js

```diff
@@ -0,0 +1,76 @@
+import React from "react";
+import { Link } from "react-router-dom";
```

Comment:
Más abajo te sugiero que use el `Link` de Material Ui, para que no colisionen los nombres tenés que cambiarlo al importarlo

```suggestion
import { Link as RouterLink } from "react-router-dom";
```